docs-v2/pages/troubleshooting.mdx

```diff
@@ -246,6 +246,18 @@ Pipedream also operates a [built-in key-value store](/data-stores/) that you can
 
 Use Pipedream's [built-in Delay actions](/workflows/flow-control/#delay) to delay a workflow at any step.
 
+### How can my workflow run faster?
+
+Here are a few things that can help your workflow execute faster:
+
+1. **Increase Memory Allocation:** Increase your [workflow memory](/workflows/settings#memory) to at least 512 MB. Raising the memory limit will proportionally increase CPU resources, leading to improved performance and reduced latency.
+
+2. **Return Static HTTP Responses:** If your workflow is triggered by an HTTP source, return a [static HTTP response](/workflows/triggers#http-responses) directly from the trigger configuration. This ensures the HTTP response is sent to the caller immediately, before the rest of the workflow steps are executed.
```

Comment:
"Return Static HTTP Responses" -> "Return static HTTP responses". Generally we only capitalize the first words of these sections, or proper nouns or other Pipedream features, but see here how "static" and "responses" should be lowercase.